README.md

```diff
@@ -6,24 +6,25 @@
 
 This project provides a number of libraries to facilitate [ConductR](http://typesafe.com/products/conductr)'s status service and its service lookup service. Note that usage of the libraries in your code is entirely benign when used outside of the context of ConductR i.e. you will find that your applications and services will continue to function normally when used without ConductR. We have also designed the libraries to be a convenience to ConductR's REST and environment variable based APIs, and to have a very low impact on your code.
 
-For the current production release of ConductR (1.0), the libraries use Akka streams/http 1.0 and are structured as follows:
-
-* `"com.typesafe.conductr" %  "conductr-bundle-lib"        % "1.0.2"`
-* `"com.typesafe.conductr" %% "scala-conductr-bundle-lib"  % "1.0.2"`
-* `"com.typesafe.conductr" %% "akka23-conductr-bundle-lib" % "1.0.2"`
-* `"com.typesafe.conductr" %% "play23-conductr-bundle-lib" % "1.0.2"`
-* `"com.typesafe.conductr" %% "play24-conductr-bundle-lib" % "1.0.2"`
-
-For ConductR 1.1 and above, the libraries use Akka streams/http 2.0 and will be:
-
-* `"com.typesafe.conductr" %  "conductr-bundle-lib"        % "1.2.0"`
-* `"com.typesafe.conductr" %% "java-conductr-bundle-lib"   % "1.2.0"`
-* `"com.typesafe.conductr" %% "scala-conductr-bundle-lib"  % "1.2.0"`
-* `"com.typesafe.conductr" %% "akka23-conductr-bundle-lib" % "1.2.0"`
-* `"com.typesafe.conductr" %% "akka24-conductr-bundle-lib" % "1.2.0"`
-* `"com.typesafe.conductr" %% "play23-conductr-bundle-lib" % "1.2.0"`
-* `"com.typesafe.conductr" %% "play24-conductr-bundle-lib" % "1.2.0"`
-* `"com.typesafe.conductr" %% "play25-conductr-bundle-lib" % "1.2.0"`
+For the current production release of ConductR (1.0), the libraries use Akka streams/http 1.x and are structured as follows:
+
+* `"com.typesafe.conductr" %  "conductr-bundle-lib"         % "1.0.2"`
```

Comment:
Changed the README accordingly with the PR https://github.com/typesafehub/conductr-lib/pull/88.